### PR TITLE
Custom FIlter Speed

### PR DESCRIPTION
**Pull request by rajamoulimallareddy**
Use Dev branch Lavalink => https://ci.fredboat.com/repository/download/Lavalink_Build/8762:id/Lavalink.jar?guest=1
for custom filter  speed it works fine

### CODE REVIEW

**Comment by rajamoulimallareddy** [comment]
> You just literally copied normal command to slash command, slash command not gonna work so fix it :/

okay

**Comment by joeyk710** [comment]
The slash commands need to be interactions. You can't send messages with interactions. Please fix the slash command area

**Review by SudhanPlayz — CHANGES_REQUESTED** [review]
You cant use `args[0]` in interaction message

File: commands/speed.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+        return client.sendTime(message.channel, `✅ | **Speed set to** \`${args[0]}\`x`);
+    },
+    SlashCommand: {
+        options: [
+            {
+                name: "speed",
+                description: `Please provide a value, value: 1-10`,
+                value: "[value]",
+                type: 3,
```
Comment (SudhanPlayz) [inline commands/speed.js:43]: type `3` is string so you better change it into number

**Review by SudhanPlayz — CHANGES_REQUESTED** [review]

File: commands/speed.js (hunk 1 above)
Comment (rajamoulimallareddy) [reply commands/speed.js:43]: okay

**Review by rajamoulimallareddy — COMMENTED** [review]
okay

**Pull request closed without merging**